Sources/CgRPC/shim/channel.c

```diff
@@ -105,8 +102,7 @@ cgrpc_call *cgrpc_channel_create_call(cgrpc_channel *channel,
                                                      NULL);
   grpc_slice_unref(host_slice);
   grpc_slice_unref(method_slice);
-  cgrpc_call *call = (cgrpc_call *) malloc(sizeof(cgrpc_call));
-  memset(call, 0, sizeof(cgrpc_call));
+  cgrpc_call *call = (cgrpc_call *) gpr_zalloc(sizeof(cgrpc_call));
```

Comment:
I assume `gpr_zalloc` is guaranteed to return zeroed memory?

Reply:
Yep, here are the docs: https://grpc.io/grpc/cpp/alloc_8h.html#ad9a7a782f4f00cad27d64d27ebbd1a72